packages/ckeditor5-theme-lark/theme/ckeditor5-table/inserttable.css

```diff
@@ -16,7 +16,7 @@
 	padding: var(--ck-insert-table-dropdown-padding) var(--ck-insert-table-dropdown-padding) 0;
 }
 
-.ck .ck-insert-table-dropdown__label {
+.ck .ck-insert-table-dropdown__label, .ck[dir=rtl] .ck-insert-table-dropdown__label {
```

Comment:
As you can see e.g. [here](https://github.com/ckeditor/ckeditor5/blob/master/docs/assets/snippet-styles.css#L37-L43), we split the comma-separated selectors into separate lines (even when there are [just two of them](https://github.com/ckeditor/ckeditor5/blob/master/packages/ckeditor5-theme-lark/theme/ckeditor5-table/tableselection.css#L30-L33)) for readability.